app/(auth)/codeStep.tsx

```diff
@@ -0,0 +1,30 @@
+import Button from "@/components/Button";
+import CustomTextInput from "@/components/CustomTextInput";
+import { AuthContext } from "@/utilities/authContext";
+import { useContext } from "react";
+import { View } from "react-native";
+import CustomText from "@/components/CustomText";
+import { useRouter } from "expo-router";
+
+export default function EmailStep() {
+  const authContext = useContext(AuthContext);
+  const router = useRouter();
+
+  return (
+    <View className="flex-1 bg-background px-xl">
+      <View className="flex-1 justify-center gap-lg py-sm">
+        <CustomText
+          style="body"
+          size="sm"
+          color="primary"
+          text="copy the code sent to your email"
```

Comment:
nit: you can make CustomText look more like the original Text component, by having it accept a `children` property. 

This way you can write: 

```
<CustomText style="body" size="sm" color="primary">copy the code sent to your email</CustomText>
```

I would also set defaults for style, color, size, so for most cases you don't have to provide them.